synapse/handlers/space_summary.py

```diff
@@ -77,13 +77,18 @@ class _PaginationKey:
 class _PaginationSession:
     """The information that is stored for pagination."""
 
+    # The time the pagination session was created, in milliseconds.
+    creatione_time_ms: int
```

Comment:
creatione! molto bene!

Reply:
Must have been my Duolingo kicking in...